src/Jaeger.Core/Senders/README.md

```diff
@@ -0,0 +1,62 @@
+# Senders
```

Comment:
nit: personally, I prefer one large readme at the root, where you can search w/ Ctrl-F

Reply:
We already had it this way with `Samplers` and `Reporters`. I usually try to avoid clutter and redundancy, but I can see your point. I will talk with some colleagues about their opinion and think about it :)

Reply:
Our department wants to keep the documents separated as it allows us to reference them in context without having to link into only a chapter of a very long document.